content/blog/2020-11-25-november-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+
+```dvc
+$ dvc checkout
+```
+
+to synchronize your `.dvc` files on that branch. But _did you know_ you can
+automate this with a `post-checkout` Git hook? We've got a hook that executes
+`dvc checkout` whenever you run `git checkout`, so you'll always have the
+correct data file versions. Head to our docs to
+[read up on installing Git hooks into your DVC repository](https://dvc.org/doc/command-reference/install#install)
+so you never forget to `dvc checkout`!
+
+### [Q: I have a big, 100 GB directory. I want to know where the contents are located so I can open them with Spark- is there a way to get the location of my files without caching them locally?](https://discord.com/channels/485586884165107732/485596304961962003/771386223403073587)
+
+For this, we'd recommend the DVC Python API's `get_url` function. For example,
+in a Python script you'd write:
+
+```python
+import dvc.api
+
+resource_url = dvc.api.get_url(
+  "<top-level-directory>",
+  repo="https://github.com/<your-repo>")
+)
+```
+
+This code means the API will return the URL for a file that ends in `.dir`. The
+`.dir` file contains a JSON-formatted table of the hashes and relative paths for
+all the files inside `<top-level-directory>`. You could then parse that file to
+tget the locations of the full directory contents.
+
+Why use the API? It's a good way to access DVC-tracked files without downloading
+them to your workspace! No downloads means nothing in your cache. Read more
+about our
+[Python API and the `get_url` function](https://dvc.org/doc/api-reference/get_url#dvcapiget_url)
+in our docs.
+
+### [Q: Can I have more than one `dvc.yaml` file in my project?](https://discord.com/channels/485586884165107732/563406153334128681/777946398250893333)
+
+_Sort of._ Here's what we mean.
+
+By default, DVC pipelines records all your stages (and their inputs and outputs)
+in a single file, `dvc.yaml`. Per directory, you can have one `dvc.yaml` file.
+If you want to run pipelines in a different folder than your project root, you
+could create another `dvc.yaml` in a subdirectory.
+
+However, `dvc.yaml` is intended to be the only file you need to record and
+reproduce pipelines per directory. Pipelines are designed to have all stages
+stored in the same place, and there's currently no method to rename `dvc.yaml`.
+
+If you really, really, _really_ want to put different stages in different files,
+there is one workaround: you can use our hidden option,
+`dvc run --single-stage`, to do this. Your stage will be stored in its own
+`.dvc` file, which can be named whatever you choose. This is a remnant from the
+pre-DVC 1.0 era. But we hope you'll embrace 1.0 and the simplicity of
+`dvc.yaml`!
+
+### [Q: How can I untrack a file that's being tracked by DVC? I want to remove it from remote storage and my local cache, too.](https://discord.com/channels/485586884165107732/563406153334128681/773277514717462548)
+
+We don't have a specific command for this, but you can do it manually:
```

Comment:
we can mention this link first - https://dvc.org/doc/user-guide/how-to/stop-tracking-data

then say, that alternative is to find the file manually (doesn't work that easy for a directory, btw), mention some link that describes how cache is organized - https://dvc.org/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory ?

Reply:
yeah, i'll mention the `dvc remove` function. Looks like you'd follow up with `dvc gc -w` according to our docs. But to get rid of the file in remote storage, like the user asked, should they use `dvc gc -w --cloud`?

Reply:
yes, it should `--cloud`. But be very careful with `-w` - it will remove a lot of data. I would just say that all `gc` commands should run with some appropriate options after that.

... or alternatively use the method you described.

Reply:
great, i've worked this in